modular-docs-manual/content/topics/module_guidelines-assembly.adoc

```diff
@@ -44,8 +44,11 @@ file3.adoc[leveloffset=+3]
 All module and assembly titles must use the H1 heading designation, such as `= My heading`.
 
 [discrete]
-== Assembly additional resources
 
-The optional additional resources list links to other material closely related to the contents of the assembly, for example, other documentation resources, instructional videos, or labs.
 
-Focus on relevant resources that might interest the user. Do not list resources for completeness.
+== Next steps and Additional resources
```

Comment:
Implement all of the decisions made regarding these sections in the procedure module template above here as well.